tianshou/trainer/offpolicy.py

```diff
@@ -117,8 +117,8 @@ def offpolicy_trainer(
                 env_step += int(result["n/st"])
                 t.update(result["n/st"])
                 logger.log_train_data(result, env_step)
-                last_rew = result['rew'] if 'rew' in result else last_rew
-                last_len = result['len'] if 'len' in result else last_len
+                last_rew = result['rew'] if result["n/ep"] > 0 else last_rew
```

Comment:
`result["n/ep"]` corresponds to episode_count in `collector.py`. If episode_count > 0, then the 'rew' and 'len' must exist.